pkg/collector/collector.go

```diff
@@ -70,11 +72,15 @@ func (b *BaseCollector) AddToCollectorFiles(file string) {
 
 // Export implements the interface method
 func (b *BaseCollector) Export() error {
-	if b.exporter != nil {
-		return b.exporter.Export(b.collectorFiles)
+	var result error
+	for _, exporter := range b.exporters {
+		if exporter != nil {
+			if err := exporter.Export(b.collectorFiles); err != nil {
+				result = multierror.Append(result, err)
+			}
+		}
```

Comment:
interface signature change to an array, no reason it had to be a single exporter